engine/src/main/java/com/google/android/fhir/Util.kt

```diff
@@ -69,5 +71,21 @@ class OffsetDateTimeTypeAdapter : TypeAdapter<OffsetDateTime>() {
   override fun read(input: JsonReader): OffsetDateTime = OffsetDateTime.parse(input.nextString())
 }
 
+/**
+ * Exclusion strategy for [Gson] that handles field exclusions for [State] returned by
+ * FhirSynchronizer. It should skip serializing the exceptions to avoid WorkManager WorkData limit
```

Comment:
```suggestion
 * Exclusion strategy for [Gson] that handles field exclusions for [State] returned by
 * FhirSynchronizer. It should skip serializing the exceptions to avoid exceeding WorkManager WorkData limit.
```

add link to issue. e.g.

`See https://github.com/google/android-fhir/issues/707.`